### DISCUSSION TITLE
Do we need a slack channel for discussions

### DISCUSSION THREAD

**Discussion by richardcase** [k3s-io/cluster-api-k3s#32] [General]
Do we need a slack/discord channel somewhere so that we can discuss the provider?

wdyt  @zawachte @mkmik @tuxtof @cannonpalms  ?

**Comment by cannonpalms** [comment]
Sure! What do you think about a channel in the Kubernetes slack? There's a rather simple process for requesting one. I believe it's one PR to update https://github.com/kubernetes/community/blob/master/communication/slack-config/channels.yaml

They have some [documentation](https://github.com/kubernetes/community/blob/master/communication/slack-guidelines.md) around the process. 

`#cluster-api-k3s`?

**Comment by tuxtof** [comment]
I agree with @cannonpalms kubernetes slack is the only choice

**Comment by richardcase** [comment]
Sounds good to me as well :+1:

**Comment by zawachte** [comment]
I can pick this up in the next few days!

**Reply by zawachte** [reply]
https://github.com/kubernetes/community/pull/7235

**Comment by zawachte** [comment]
https://kubernetes.slack.com/archives/C051K6ADDQB/p1680542031116119
